### ISSUE TITLE
Train on music

### ISSUE THREAD

**Issue by venkatasg** [venkatasg/Lil-Bevo#10]
Dataset 1: [Maestro](https://magenta.tensorflow.org/datasets/maestro) is what Isabel used in her paper. Gonna start there. How do I train a tokenizer on MIDI files? Do I train a new tokenizer on both our dataset plus the MIDI files, or use the existing tokenizer on the MIDI files?

[Isabel's paper](https://arxiv.org/pdf/2004.14601.pdf)

**Comment by venkatasg** [comment]
Trained on MAESTRO with a tokenizer trained on 10M tokens + MAESTRO. Small, but consistent improvement across the board in BLiMP scores.
<img width="944" alt="Screenshot 2023-06-04 at 13 10 25" src="https://github.com/venkatasg/Lil-Bevo/assets/22871413/cd7d8c21-e889-4ee6-afca-04bdb27407bd">

**Comment by venkatasg** [comment]
Updated results, after OPT was trained fully. Additionally, I took the checkpoint where eval loss stabilized, and continued training on a training set of 10M tokens plus MAESTRO data. This training setup is more stable than suddenly changing training data.

<img width="859" alt="Screenshot 2023-06-13 at 22 18 27" src="https://github.com/venkatasg/Lil-Bevo/assets/22871413/aaafe921-ef44-43c7-9468-4d3fb7ab6bb6">

I also tried training with random sampling of music. Similar results. Music doesn't seem to hurt, but doesn't really help with loss or BLiMP results.
